cpp/src/securemessage/crypto_ops_openssl.cc

```diff
@@ -19,6 +19,8 @@
 #include <openssl/ec.h>
```

Comment:
This file merged a bit messy.  Can you confirm that this branch builds and runs all the tests (and that they pass)?

Reply:
this big-ish diff is the result of internal review. I verified it then (at the time of writing) and verified it again now. Unit tests are passing.